Codecademy/gamut
# feat(ConnectedForm): pass onError prop through to react-hook-form

### Overview

~still need to test this - `yarn link` gives me all kinds of trouble (I think because some of my configs are symlinked) so I'm just gonna test with the alpha~

[edit] it works!





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/Ym03UHdFa0ctNFVoMndKze/Plan-Split%3A-pricing-and-payment-v2?node-id=1228%3A86569
- [x] Related to JIRA ticket: [GRW-203](https://codecademy.atlassian.net/browse/GRW-203)

#### Testing Instructions



- pull down alpha
- pass `onError` prop
- ensure it's called when form validation fails

#### PR Links and Envs

[portal-app](https://github.com/codecademy-engineering/portal-app/pull/2438)

testing the above is a bit of a pain
- locally create a test user and check "include this user in the optimizely 'Tester' audience"
- verify their email and impersonate
- go to `/checkout?product=pro-gold`
- submit the checkout form in a variety of incorrect ways and ensure the error message reflects the invalid fields
